src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/SQLFunctionsIT.java

```diff
@@ -0,0 +1,217 @@
+//        System.out.println(searchDao.explain(query).explain().explain());
+//    }
+
+// todo: change when split is back on language
+//    @Test
+//    public void whereConditionLeftFunctionRightVariableEqualTest() throws Exception {
+//
+//        String query = "SELECT " +
+//                " * from " +
+//                TestsConstants.TEST_INDEX + "/account " +
+//                " where split(address,' ')[0]='806' limit 1000  ";
+//
+//        CSVResult csvResult = getCsvResult(false, query);
+//        List<String> contents = csvResult.getLines();
+//        Assert.assertTrue(contents.size() == 4);
+//    }
+//
+//    @Test
+//    public void whereConditionLeftFunctionRightVariableGreatTest() throws Exception {
+//
+//        String query = "SELECT " +
+//                " * from " +
+//                TestsConstants.TEST_INDEX + "/account " +
+//                " where floor(split(address,' ')[0]+0) > 805 limit 1000  ";
+//
+//        SearchDao searchDao = MainTestSuite.getSearchDao() != null ? MainTestSuite.getSearchDao() : getSearchDao();
+//        System.out.println(searchDao.explain(query).explain().explain());
+//
+//        CSVResult csvResult = getCsvResult(false, query);
+//        List<String> contents = csvResult.getLines();
+//        Assert.assertTrue(contents.size() == 223);
+//    }
+
+    @Test
+    public void concat_ws_fields() throws Exception {
+
+        //here is a bug,csv field with spa
+        String query = "SELECT " +
+                " concat_ws('-',age,address) as combine,address from " +
+                TEST_INDEX_ACCOUNT + "/account " +
+                " limit 10  ";
+        assertThat(
+                executeQuery(query),
+                hitAny(kvString("/fields/combine/0", containsString("-")))
+        );
+    }
+
+    @Test
+    public void functionLogs() throws Exception {
+        String query = "SELECT log10(100) as a, log(1) as b, log(2, 4) as c, log2(8) as d from "
+                + TEST_INDEX_ACCOUNT + "/account limit 1";
+
+        assertThat(
+                executeQuery(query),
+                hitAny(both(kvDouble("/fields/a/0", equalTo(2.0)))
+                        .and(kvDouble("/fields/b/0", equalTo(0.0)))
+                        .and(kvDouble("/fields/c/0", equalTo(1.0)))
+                        .and(kvDouble("/fields/d/0", equalTo(3.0))))
+        );
+    }
+
+    @Test
+    public void functionPow() throws Exception {
+        String query = "SELECT pow(account_number, 2) as key,"+
+                "abs(age - 60) as new_age from " + TEST_INDEX_ACCOUNT + "/account WHERE firstname = 'Virginia' and lastname='Ayala' limit 1";
+
+        assertThat(
+                executeQuery(query),
+                hitAny(both(kvDouble("/fields/new_age/0", equalTo(21.0))).and(kvDouble("/fields/key/0", equalTo(625.0))))
+        );
+    }
+
+//    // todo: change when split is back on language
```

Comment:
same here, I suggest uncommenting and adding `@Ignore`

Reply:
Done.